Src/FluentAssertions/Execution/LateBoundTestFramework.cs

```diff
@@ -16,8 +16,10 @@ public void Throw(string message)
 
         if (exceptionType is null)
         {
+#pragma warning disable CA2201 // Do not raise reserved exception types
```

Comment:
I kind of agree with this analyzer, but not sure which exception type we should throw instead 🤔

Reply:
`NotSupportedException`?